.github/workflows/app-distribute.yml

```diff
@@ -0,0 +1,39 @@
+name: App Distribute CI
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build_virtualevent_sample_app:
+    name: Build and Distribute VirtualEvent Sample App
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          distribution: adopt
+          java-version: 11
+      - name: Prepare environment
+        run: |
+          git fetch --unshallow
+          echo "${{ secrets.RELEASE_KEYSTORE }}" > .sign/release.keystore.asc
+          gpg -d --passphrase "${{ secrets.PASSPHRASE }}" --batch .sign/release.keystore.asc > .sign/release.keystore
+          echo "${{ secrets.RELEASE_KEYSTORE_PROPERTIES }}" > .sign/keystore.properties.asc
+          gpg -d --passphrase "${{ secrets.PASSPHRASE }}" --batch .sign/keystore.properties.asc > .sign/keystore.properties
+      - name: Assemble
+        run: bash ./gradlew :virtualevent-demo:assembleRelease --stacktrace
```

Comment:
Yes, I agree but that's how we have it in our sample app. Not sure what was the reason for doing so.

Reply:
🤔  It is choosing `bash` as the shell script. But, if I understand this right, when you use `./` to execute, you let it choose the default shell script. Ubuntu already uses bash as the default: https://askubuntu.com/questions/87853/what-is-default-shell-for-terminal, so it doesn't make difference. I personally prefer to let the image uses its default as it may change with time... but fell free to use what you prefer.